R/format.R

```diff
@@ -647,6 +647,6 @@ format.parameters_sem <- function(x,
       msg <- paste(msg, "Uncertainty intervals for random effect variances computed using a Wald z-distribution approximation.")
     }
 
-    message(insight::format_message(msg))
+    message(insight::format_message("\033[0;33m", msg, "\033[0m"))
```

Comment:
I suggest reverting this change, or at least use `insight::color_text()`, which checks if ANSI-colors are available.

Reply:
I revert it for now and we can also make another PR later to change the color if needed